modules/SofaGeneralLoader/SphereLoader.h

```diff
@@ -21,11 +21,8 @@
 ******************************************************************************/
 #ifndef SOFA_COMPONENT_LOADER_SPHERELOADER_H
 #define SOFA_COMPONENT_LOADER_SPHERELOADER_H
-#include "config.h"
```

Comment:
I'm not sure about removing this one....because no one will define SOFA_TARGET anymore. 

While it is supposed to be used in the ObjectFactory:
```cpp
/// The name of the library or executable containing the binary code for this component
    virtual const char* getTarget()
    {
#ifdef SOFA_TARGET
        return sofa_tostring(SOFA_TARGET);
#else
        return "";
#endif
    }
```

NB: I'm really not found of silently ignoring the undefined SOFA_TARGET.

Reply:
Maybe we should replace this code by:
```cpp
#ifndef SOFA_TARGET
        #error....the SOFA_TARGET is not defined. 
#else: 
        return sofa_tostring(SOFA_TARGET);
#endif 
```
(Not in this PR of course)

Reply:
I will report our discussion in an issue and remove that change from the PR to have only changes on header includes.